pkg/framework/context/context.go

```diff
@@ -13,12 +13,14 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
 	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api"
+	"github.com/hyperledger/aries-framework-go/pkg/wallet"
 )
 
 // Provider supplies the framework configuration to client objects.
 type Provider struct {
 	outboundTransport transport.OutboundTransport
 	services          []dispatcher.Service
+	wallet            wallet.Wallet
```

Comment:
How is wallet intended to interact with store?

Reply:
done

Reply:
Any more context? :)